include/libtorrent/file.hpp

```diff
@@ -187,6 +187,14 @@ namespace libtorrent
 // internal export should be used at unit tests only
 	TORRENT_EXTRA_EXPORT native_path_string convert_to_native_path_string(std::string const& path);
 
+// internal export should be used at unit tests only
+	TORRENT_EXTRA_EXPORT std::string convert_from_native_path(char const* s);
```

Comment:
if ``TORRENT_USE_WSTRING`` is defined, is this function used? it feels like these functions should be mutually exclusive